cmake/alpakaCommon.cmake

```diff
@@ -18,7 +19,10 @@ set(ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE_DEFAULT ON)
 set(ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE_DEFAULT ON)
 set(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE_DEFAULT ON)
 set(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE_DEFAULT ON)
-set(ALPAKA_ACC_CPU_BT_OMP4_ENABLE_DEFAULT ON)
+set(ALPAKA_ACC_ANY_BT_OMP5_ENABLE_DEFAULT ON)
+set(ALPAKA_ACC_ANY_BT_OACC_ENABLE_DEFAULT OFF)
```

Comment:
I think this should not be part of this PR.

Reply:
fixed